packages/website/src/components/ApplyForm.js

```diff
@@ -165,11 +165,11 @@ function ApplyForm() {
           <Input
             required={true}
             id="repoURL"
-            type="repoURL"
+            type="url"
```

Comment:
Ah yes make sense, I realized there was some weirdly formatted ones on the hub 🤔

Reply:
yup a lot, I wasn't sure it was intentional but a few people were taking the PIPE very seriously. Should remove some error 👍🏻

Reply:
good catch!